README.md

```diff
@@ -16,33 +15,46 @@ Major features:
+
+As a last resort, submit a [help request](https://github.com/ckb-next/ckb-next/issues/new/choose) via an issue. 
+
+## Contribute
+Check out our [Contributing Guidelines](https://github.com/ckb-next/ckb-next/wiki/Contributing) before submitting new issues.
+
+### [Report a Bug](https://github.com/ckb-next/ckb-next/issues/new/choose) select "Bug Report"
+
+### [Request an Enhancement](https://github.com/ckb-next/ckb-next/issues/new/choose) select "Featrue Request"
+
+### [Request a new Device](https://github.com/ckb-next/ckb-next/issues/new/choose) select "New Device Request."
 
-### Important information regarding macOS
-macOS is no longer officially supported. For more information, please refer to [issue #660](https://github.com/ckb-next/ckb-next/issues/660).
 
-### General information
 
-Most of the information can be found on [ckb-next wiki pages](https://github.com/ckb-next/ckb-next/wiki).
+> Maintainers reserve the right to modify and remove issues, pull requests, and comments that are denunciating, off-topic, harmful, hateful, and inappropriate.
+Please be appreciative, humble, and kind to each other.
 
-[Supported devices](https://github.com/ckb-next/ckb-next/wiki/Supported-Hardware).
+## General information
 
-[Linux Installation](https://github.com/ckb-next/ckb-next/wiki/Linux-Installation).
+You can find most of the information in the [ckb-next wiki pages](https://github.com/ckb-next/ckb-next/wiki).
 
-[Troubleshooting](https://github.com/ckb-next/ckb-next/wiki/Troubleshooting).
+### [Installation](https://github.com/ckb-next/ckb-next/wiki/Linux-Installation).
 
-[Known Issues](https://github.com/ckb-next/ckb-next/wiki/Known-issues).
+### [Troubleshooting](https://github.com/ckb-next/ckb-next/wiki/Troubleshooting).
 
-[Contributing](https://github.com/ckb-next/ckb-next/wiki/Contributing).
+### [Known Issues](https://github.com/ckb-next/ckb-next/wiki/Known-issues).
 
-[Community Pipe Animation Scripts](https://github.com/ckb-next/ckb-next/wiki/Community-Pipe-Scripts).
+### [Community Pipe Animation Scripts](https://github.com/ckb-next/ckb-next/wiki/Community-Pipe-Scripts).
 
-### Contact
 
-Maintainers reserve the rights to modify and remove issues, pull requests and comments therein, that are denunciating, off-topic, harmful, hateful and overall inappropriate.
-Please be appreciative, humble and kind to each other.
 
-* IRC chat: `#ckb-next` channel at [irc.libera.chat](https://web.libera.chat/?channels=#ckb-next)
-* [Matrix](https://matrix.to/#/#ckb-next:matrix.org): `#ckb-next:matrix.org`
-* Mailing list: [ckb-next@googlegroups.com](https://groups.google.com/forum/#!forum/ckb-next)
-* [GitHub Issues](https://github.com/ckb-next/ckb-next/issues)
+***Disclaimer:*** ckb-next is not an official Corsair product. It is not supported by, affiliated with, or endorsed by Corsair. It is licensed under the GNU General Public License (version 2) in the hope that it will be helpful, but with NO WARRANTY of any kind.
```

Comment:
The disclaimer was put near the top of the readme for a reason, please do not bury it.

Reply:
It will be moved back in my next push.